serfclient/connection.py

```diff
@@ -107,6 +107,14 @@ def handshake(self):
             self._socket = self._connect()
         return self.call('handshake', {"Version": 1}, expect_body=False)
 
+    def auth(self, auth_key):
```

Comment:
Probably doesn't need to be public; I could throw an underscore on there. Doesn't the same apply to `handshake`, which basically was what I copied from?